### PR TITLE
Updated UserMSVC

### PR DESCRIPTION
**Pull request by warkionw**
Updated responses to fit Unified Response Format, Cleaned code, Updated README with new responses, Made new UnauthorizedException

### CODE REVIEW

File: Backend/User/README.md [hunk 1]
```diff
@@ -59,20 +66,34 @@ Sample Success 200 Response:
...
+Sample Failed 401 Response:
+
+```json
+{
+    "success": false,
+    "message": "Invalid email or password",
+    "content": null
+}```
+
 ### POST /logout
```
Comment (ZhiLinR) [inline Backend/User/README.md:97]: somethin happened here

File: Backend/User/src/main/java/user/controller/ProfileController.java
Comment (ZhiLinR) [inline Backend/User/src/main/java/user/controller/ProfileController.java:1]: looks okay

File: Backend/User/src/main/java/user/util/ResponseManager.java
Comment (ZhiLinR) [inline Backend/User/src/main/java/user/util/ResponseManager.java:1]: more or less lgtm

File: Backend/User/src/main/java/user/controller/ProfileController.java
Comment (warkionw) [reply Backend/User/src/main/java/user/controller/ProfileController.java:1]: tq boss i really appreciate your work

File: Backend/User/src/main/java/user/util/ResponseManager.java
Comment (warkionw) [reply Backend/User/src/main/java/user/util/ResponseManager.java:1]: :0 amazing boss

**Review by ZhiLinR — APPROVED** [review]
just the readme

**Review by ZhiLinR — APPROVED** [review]
nice

**Comment by ZhiLinR** [comment]
ResponseManager can be possibly reused by @arshiya8  for the unified response format

**Pull request merged**